src/components/structures/RoomView.js

```diff
@@ -45,6 +45,8 @@ import RoomViewStore from '../../stores/RoomViewStore';
 import RoomScrollStateStore from '../../stores/RoomScrollStateStore';
 import SettingsStore from "../../settings/SettingsStore";
 
+import _every from "lodash/every";
+
```

Comment:
any chance you could use the `Array.prototype.every` instead of lodash? They look identical to me

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/every